backend/dal/Migrations/v01.05.06/Up/PostUp/01-BuildingPredominateUses.sql

```diff
@@ -0,0 +1,13 @@
+PRINT 'Add Marina to Building Predominate Uses'
+
+--Add Marina
+
+INSERT INTO dbo.[BuildingPredominateUses] (
```

Comment:
This will update production and any database that is currently setup.  However it will break for any database refresh or the initial setup.  This is because you update the `init` migration to already include this.  When the migration runs this v01.05.06 it will throw an error as the database already has the value.  You need to first make sure there is no Marina value and then insert.

Reply:
You will need to handle rollback and roll forward.  For example, if I apply this migration it will add the new value `Marina`.  Then when I rollback it will remain.  That means if I reapply (roll forward again), it will attempt to re-add the `Marina` and result in a failure.

Reply:
will this change work now?